packages/fluentui/react-builder/src/components/Designer.tsx

```diff
@@ -55,6 +55,9 @@ type DesignerState = {
   codeError: string | null;
 };
 
+const history = [];
```

Comment:
What is the motivation for having `history` and `redo` as global variables?
I would rather have them as part of the state. Whenever those are changed, the component is rerendered anyway.